content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Setup Repo Info
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
+
+*See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
+
+## Installing the Components
+
+### Installing Base Component
+
+The **Base** chart contains the basic CRDs and cluster roles required to set up Istio.
+This should be installed prior to any other Istio component.
+
+{{< text bash >}}
+$ helm install istio-base istio/base
+{{< /text >}}
+
+### Installing CNI Component
+
+The **CNI** chart installs the Istio CNI Plugin. There are some main roles of Istio CNI Plugin:
+
+1. Eliminates the need for the `istio-init` container that sets up traffic routing for sidecar proxies.
+
+1. In Ambient, it is responsible for detecting the pods that belong to the ambient mesh, and configuring 
```

Comment:
Remove trailing space to fix lint
```suggestion
1. In Ambient, it is responsible for detecting the pods that belong to the ambient mesh, and configuring
```

Reply:
Thanks, I was trying to figure out where's the trailing space :)